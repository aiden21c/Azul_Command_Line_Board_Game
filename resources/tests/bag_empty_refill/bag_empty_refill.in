2
bag_empty_refill.txt
turn 0 R 5
save bag_empty_refill_actual.out