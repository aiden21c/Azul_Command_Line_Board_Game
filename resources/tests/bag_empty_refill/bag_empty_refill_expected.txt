72
0
John
9
Kate
10
F                
RRYU
RRLU
YYBU
YYBB
RRYB
.
..
...
.UUU
.....

bYrUlLByruuLbyRrulbyyrulb
.
..
...
....
.....

ByRulLByruulbyRrulbYyrulb

LYUULLBYBRRBBRULBYRYYLBBLULBUBYLURLLLLRRRRYY